schema/all_test.go

```diff
@@ -56,6 +56,34 @@ func (tc fieldValidatorTestCase) Run(t *testing.T) {
 	})
 }
 
+type fieldSerializerTestCase struct {
+	Name             string
+	Serializer       schema.FieldSerializer
+	ReferenceChecker schema.ReferenceChecker
+	Input, Expect    interface{}
+	Error            string
+}
+
+func (tc fieldSerializerTestCase) Run(t *testing.T) {
+	t.Run(tc.Name, func(t *testing.T) {
+		t.Parallel()
+
+		if cmp, ok := tc.Serializer.(schema.Compiler); ok {
+			err := cmp.Compile(tc.ReferenceChecker)
+			assert.NoError(t, err, "Validator.Compile(%v)", tc.ReferenceChecker)
```

Comment:
Sorry for not pointing this out earlier. I realize this is copy-paste of the existing fieldValidatorTestCase, but we are trying to move away from using testify in Clarify. There is a short discussion about it in #169.

In short, it would be nice (perhaps as a separate commit) to rewrite these tests to standard if checks and calls to `t.Errorf` or `t.Error` on errors.